.github/workflows/pr-ubuntu-docker.yml

```diff
@@ -72,24 +72,18 @@ jobs:
                 options: |
                     -v ${{github.workspace}}:/src
                     -u skymp
-                    --name configure_container
                 run: |
                     cd /src \
                     && ./build.sh --configure \
                        -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} \
                        -DUNIT_DATA_DIR="/src/skyrim_data_files"
 
-          - name: Copy log file from container to workspace
-            if: failure()
-            run: |
-              sudo docker cp configure_container:/src/vcpkg/buildtrees/rsm-bsa/install-x64-linux-dbg-out.log ${{github.workspace}}/install-x64-linux-dbg-out.log
```

Comment:
No need, the whole workspace is mounted with `-v ${{github.workspace}}:/src`

See also https://github.com/skyrim-multiplayer/skymp/actions/runs/12456833845?pr=2275